src/components/TeamFounder/FounderSection.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { memo } from "react";
+import useViewport from "../UseViewport";
+import * as S from "../TeamSection/TeamSection.styles";
+import { founder } from "./FounderInformation";
+import Founder from "./Founder";
+
+
+const FounderSection = () => {
```

Comment:
I don't think there's any need for this parent component for `Founder`, all you're doing is passing props down to the child component. We can just set the width and get the founder info directly in `Founder` instead.

Reply:
Once you remove this you can rename `Founder` to `FounderSection`